src/components/dialogs/network-modifications/by-filter/by-assignment/assignment/assignment-form.tsx

```diff
@@ -0,0 +1,158 @@
+    if (prevDataType && prevDataType !== dataType) {
+        setValue(`${name}.${index}.${VALUE_FIELD}`, dataType === DataType.BOOLEAN ? false : null);
+    }
+
+    const filtersField = (
+        <DirectoryItemsInput
+            name={`${name}.${index}.${FILTERS}`}
+            equipmentTypes={[equipmentType]}
+            elementType={ElementType.FILTER}
+            label={'filter'}
+            titleId={'FiltersListsSelection'}
+            disable={!equipmentType}
+        />
+    );
+
+    const editedField = (
+        <AutocompleteInput
+            name={`${name}.${index}.${EDITED_FIELD}`}
+            options={equipmentFields}
+            label={'EditedField'}
+            size={'small'}
+            inputTransform={(value: any) => equipmentFields.find((option) => option?.id === value) || value}
+            outputTransform={(option: any) => getIdOrValue(option) ?? null}
+            getOptionLabel={(option: any) => (option ? intl.formatMessage({ id: getLabelOrValue(option) }) : option)}
+            isOptionEqualToValue={areIdsEqual}
+        />
+    );
+
+    const propertyNameField = (
+        <AutocompleteInput
+            name={`${name}.${index}.${PROPERTY_NAME_FIELD}`}
+            options={predefinedPropertiesNames}
+            label={'PropertyName'}
+            size={'small'}
+            allowNewValue
+        />
+    );
+
+    const valueField = useMemo(() => {
+        if (dataType === DataType.PROPERTY) {
+            return (
+                <AutocompleteInput
+                    name={`${name}.${index}.${VALUE_FIELD}`}
+                    label={'PropertyValue'}
+                    options={predefinedPropertiesValues}
+                    size={'small'}
+                    allowNewValue
+                />
+            );
+        }
+
+        if (dataType === DataType.INTEGER) {
+            return <IntegerInput name={`${name}.${index}.${VALUE_FIELD}`} label={'Value'} />;
+        }
+
+        if (dataType === DataType.BOOLEAN) {
+            return <SwitchInput name={`${name}.${index}.${VALUE_FIELD}`} formProps={{ value: false }} />;
+        }
+
+        if (dataType === DataType.ENUM) {
+            return (
+                <SelectInput name={`${name}.${index}.${VALUE_FIELD}`} label="Value" options={options} size={'small'} />
+            );
+        }
+
+        // by default is a numeric type
+        return <FloatInput name={`${name}.${index}.${VALUE_FIELD}`} label={'Value'} />;
+    }, [dataType, name, index, predefinedPropertiesValues, options]);
+
+    return (
+        <>
+            {gridItem(filtersField, 2.25)}
+            {gridItem(editedField, 3)}
+            {dataType === DataType.PROPERTY && (
+                <>
+                    {gridItem(propertyNameField, 2.0)}
+                    {gridItem(<DensityLargeIcon fontSize="small" sx={{ marginTop: 1 }} />, 0.25)}
+                </>
+            )}
+            {gridItem(valueField, dataType === DataType.PROPERTY ? 3.25 : 5.5)}
```

Comment:
Why is filter field so small even if it may contain several filters ? I think edited field and maybe value field should be smaller than the filter field, just like in the PO ticket.

Reply:
DONE, I make filter field larger a bit